seatunnel-connectors-v2/connector-file/connector-file-oss/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/oss/fs/OSSFileSystem.java

```diff
@@ -0,0 +1,997 @@
+            ossConf.setProxyUsername(proxyUsername);
+        }
+        if (proxyPassword != null) {
+            ossConf.setProxyPassword(proxyPassword);
+        }
+        ossConf.setProxyDomain(conf.getTrimmed(HADOOP_PROXY_DOMAIN));
+        ossConf.setProxyWorkstation(conf.getTrimmed(HADOOP_PROXY_WORKSTATION));
+        //MaxConnections
+        ossConf.setMaxConnections(conf.getInt(HADOOP_MAXIMUM_CONNECTIONS, DEFAULT_MAXIMUM_CONNECTIONS));
+        //SocketTimeout
+        ossConf.setSocketTimeout(conf.getInt(HADOOP_SOCKET_TIMEOUT, DEFAULT_SOCKET_TIMEOUT));
+        //ConnectionTimeout
+        ossConf.setConnectionTimeout(conf.getInt(HADOOP_ESTABLISH_TIMEOUT, DEFAULT_ESTABLISH_TIMEOUT));
+        //MaxErrorRetry
+        ossConf.setMaxErrorRetry(conf.getInt(HADOOP_MAX_ERROR_RETRIES, DEFAULT_MAX_ERROR_RETRIES));
+        //Protocol
+        boolean secureConnections = conf.getBoolean(HADOOP_SECURE_CONNECTIONS, DEFAULT_SECURE_CONNECTIONS);
+        ossConf.setProtocol(secureConnections ? Protocol.HTTPS : Protocol.HTTP);
+        //SupportCname
+        ossConf.setSupportCname(false);  //ListBuckets can not be used when enable CNAME.
+        ossConf.setSLDEnabled(false);
+
+        //extra configuration for multiple part copy/upload
+        ossConf.setMultipartUploadThreshold(conf.getLong(HADOOP_MULTIPART_UPLOAD_THRESHOLD, DEFAULT_MULTIPART_UPLOAD_THRESHOLD));
+        ossConf.setMinimumUploadPartSize(conf.getLong(HADOOP_MULTIPART_UPLOAD_PART_SIZE, DEFAULT_MINIMUM_UPLOAD_PART_SIZE));
+        ossConf.setMultipartCopyThreshold(conf.getLong(HADOOP_MULTIPART_COPY_THRESHOLD, DEFAULT_MULTIPART_COPY_THRESHOLD));
+        ossConf.setMultipartCopyPartSize(conf.getLong(HADOOP_MULTIPART_COPY_PART_SIZE, DEFAULT_MINIMUM_COPY_PART_SIZE));
+        // extra configuration for multiple part copy/upload thread pool
+        ossConf.setCorePoolSize(conf.getInt(HADOOP_CORE_POOL_SIZE, DEFAULT_CORE_POOL_SIZE));
+        ossConf.setMaxPoolSize(conf.getInt(HADOOP_MAX_POOL_SIZE, DEFAULT_MAX_POOL_SIZE));
+        ossConf.setKeepAliveTime(conf.getInt(HADOOP_KEEP_ALIVE_TIME, DEFAULT_KEEP_ALIVE_TIME));
+
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Using proxy server {}:{} as user {} with password {} on " +
+                            "domain {} as workstation {}", ossConf.getProxyHost(),
+                    ossConf.getProxyPort(), ossConf.getProxyUsername(),
+                    ossConf.getProxyPassword(), ossConf.getProxyDomain(),
+                    ossConf.getProxyWorkstation());
+        }
+
+        client = new SmartOSSClient(endpoint, accessKeyId, accessKeySecret, ossConf);
+        if (!client.doesBucketExist(bucket)) {
+            throw new IOException("Bucket " + bucket + " does not exist");
+        }
+        setConf(conf);
+
+    }
+
+    /**
+     * Return the protocol scheme for the FileSystem.
+     *
+     * @return "oss"
+     */
+    public String getScheme() {
+        return "oss";
+    }
```

Comment:
this file has a lot of Missing '@Override' annotation on some method,please check again.